src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataServiceSpec.scala

```diff
@@ -94,121 +93,6 @@ class FileMetadataServiceSpec extends AnyFlatSpec with MockitoSugar with Matcher
     result.value should equal(value)
   }
 
-  "updateBulkFileMetadata" should "delete existing metadata rows and add new metadata rows based on the input" in {
```

Comment:
Unit tests should remain in place until the methods are removed. The deleted unit tests all related to methods that will be removed in subsequent PRs. They should be deleted in those PRs, not this one.